environments/gcp/flyte-core/values-gcp-core.yaml

```diff
@@ -0,0 +1,391 @@
+userSettings:
+  googleProjectId: ${gcp-project-id}
+  dbHost: ${dbhost}
+  dbPassword: ${dbpassword}
+  bucketName: ${gcsbucket}
+  hostName: ${hostname}
+  flyteadminServiceAccount: ${flyteadminServiceAccount}
+  flytepropellerServiceAccount: ${flytepropellerServiceAccount}
+  flyteschedulerServiceAccount: ${flyteschedulerServiceAccount}
+  datacatalogServiceAccount: ${datacatalogServiceAccount}
+  flyteworkersServiceAccount: ${flyteworkersServiceAccount}
+
+#
+# FLYTEADMIN
+#
+
+flyteadmin:
+  replicaCount: 1
+  serviceAccount:
+    # -- If the service account is created by you, make this false, else a new service account will be created and the flyteadmin role will be added
+    # you can change the name of this role
+    create: true
+    annotations:
+      # Needed for gcp workload identity to function
+      # https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+      iam.gke.io/gcp-service-account: "{{ .Values.userSettings.flyteadminServiceAccount }}@{{ .Values.userSettings.googleProjectId }}.iam.gserviceaccount.com"
+  resources:
+    limits:
+      ephemeral-storage: 200Mi
+    requests:
+      cpu: 50m
+      ephemeral-storage: 200Mi
+      memory: 200Mi
+  service:
+    annotations:
+      # Required for the ingress to properly route grpc traffic to grpc port
+      cloud.google.com/app-protocols: '{"grpc":"HTTP2"}'
```

Comment:
Are you sure this is required for the nginx ingress? As far as I know, this is only needed for the gce ingress.